cmd/juju/commands/bootstrap.go

```diff
@@ -145,6 +167,12 @@ Examples:
     juju bootstrap --agent-version=2.2.4 aws joe-us-east-1
     juju bootstrap --config bootstrap-timeout=1200 azure joe-eastus
 
+    # For a bootstrap on k8s, setting the service type of the Juju controller service to LoadBalancer
+    juju bootstrap --config controller-service-type=LoadBalancer
+
+	# For a bootstrap on k8s, setting the service type of the Juju controller service to External
+    juju bootstrap --config controller-service-type=ExternalName,controller-service-name=controller.juju.is
```

Comment:
This is also not the actual way that multiple config sections are given. I tried with a comma and I tried with quotes around a space. What worked best was:
juju bootstrap --config controller-service-type=externalname --config controller-service-name=controller.juju.is

Reply:
Fixed